plugins/calendar/src/main/java/de/unistuttgart/iaas/amyassist/amy/plugin/calendar/CalendarEvent.java

```diff
@@ -110,126 +141,106 @@ public LocalDateTime getStart() {
 		return this.start;
 	}
 
-	/**
-	 * Set's {@link #start start}
-	 * 
-	 * @param start
-	 *            start
-	 */
-	public void setStart(LocalDateTime start) {
-		this.start = start;
-	}
-
 	/**
 	 * Get's {@link #end end}
-	 * 
+	 *
 	 * @return end
 	 */
 	public LocalDateTime getEnd() {
 		return this.end;
 	}
 
-	/**
-	 * Set's {@link #end end}
-	 * 
-	 * @param end
-	 *            end
-	 */
-	public void setEnd(LocalDateTime end) {
-		this.end = end;
-	}
-
 	/**
 	 * Get's {@link #summary summary}
-	 * 
+	 *
 	 * @return summary
 	 */
 	public String getSummary() {
 		return this.summary;
 	}
 
-	/**
-	 * Set's {@link #summary summary}
-	 * 
-	 * @param summary
-	 *            summary
-	 */
-	public void setSummary(String summary) {
-		this.summary = summary;
-	}
-
 	/**
 	 * Get's {@link #location location}
-	 * 
+	 *
 	 * @return location
 	 */
 	public String getLocation() {
 		return this.location;
 	}
 
-	/**
-	 * Set's {@link #location location}
-	 * 
-	 * @param location
-	 *            location
-	 */
-	public void setLocation(String location) {
-		this.location = location;
-	}
-
 	/**
 	 * Get's {@link #description description}
-	 * 
+	 *
 	 * @return description
 	 */
 	public String getDescription() {
 		return this.description;
 	}
 
-	/**
-	 * Set's {@link #description description}
-	 * 
-	 * @param description
-	 *            description
-	 */
-	public void setDescription(String description) {
-		this.description = description;
-	}
-
 	/**
 	 * Get's {@link #allDay allDay}
-	 * 
+	 *
 	 * @return allDay
 	 */
 	public boolean isAllDay() {
 		return this.allDay;
 	}
 
 	/**
-	 * Set's {@link #allDay allDay}
-	 * 
-	 * @param allDay
-	 *            allDay
+	 *
+	 * @return the recurrence options
 	 */
-	public void setAllDay(boolean allDay) {
-		this.allDay = allDay;
+	public String getRecurrence() {
+		return this.recurrence;
+	}
+
+	/**
+	 *
+	 * @return the reminder type
+	 */
+	public String getReminderType() {
+		return this.reminderType;
+	}
+
+	/**
+	 *
+	 * @return the reminder time in minutes
+	 */
+	public int getReminderTime() {
+		return this.reminderTime;
+	}
+
+	/**
+	 *
+	 * @return the reminders of the Event as Event.Reminders from google api
+	 */
+	public Event.Reminders getReminders() {
+		EventReminder[] reminderOverrides = new EventReminder[] {
+				new EventReminder().setMethod(this.reminderType).setMinutes(this.reminderTime), };
+
+		return new Event.Reminders().setUseDefault(false).setOverrides(Arrays.asList(reminderOverrides));
 	}
 
 	@Override
 	public boolean equals(Object o) {
-		if (this == o)
```

Comment:
If you need this for testing, do not use it because it makes no sense to rely on a method of a class you are testing. Export your booleans to your test line by line like following:
`assertThat(event1.id, is(event2.id));`
`assertThat(event1.start, is(event2.start));`
`...`

Reply:
needed for REST Test